frontend/src/features/admin/edit-user/EditUserPage.tsx

```diff
@@ -91,20 +91,20 @@ const EditUserPage = (props: IEditUserPageProps) => {
   };
 
   const initialValues = {
-    username: user.username,
-    firstName: user.firstName,
-    lastName: user.lastName,
-    email: user.email,
-    displayName: user.displayName,
-    isDisabled: !!user.isDisabled,
-    rowVersion: user.rowVersion,
+    username: user.username ?? '',
```

Comment:
handle undefined values to prevent formik warnings when loading an existing user.

Reply:
This is where I'd probably use `{...defaultValues, ...initialValues}`.